frontend/src/components/menu/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { ReactNode } from 'react';
+import { ListGroup, Form } from 'react-bootstrap';
+import classNames from 'classnames';
+import ClickAwayListener from 'react-click-away-listener';
+import './menu.scss';
```

Comment:
Minor comment on naming. We have adopted to name component files with Uppercase - i.e. `Menu.tsx` and `Menu.scss` vs `menu.tsx`.

We also have the folder start with Uppercase to indicate this is a component.